rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1229,19 +1359,19 @@ private Integer commitFederation(boolean dryRun, Sha3Hash hash) throws IOExcepti
             return 1;
         }
 
-        // Move UTXOs from the active federation into the retiring federation
-        // and clear the active federation's UTXOs
-        List<UTXO> utxosToMove = new ArrayList<>(provider.getActiveFederationBtcUTXOs());
-        provider.getActiveFederationBtcUTXOs().clear();
-        List<UTXO> retiringFederationUTXOs = provider.getRetiringFederationBtcUTXOs();
-        retiringFederationUTXOs.clear();
-        utxosToMove.forEach(utxo -> retiringFederationUTXOs.add(utxo));
+        // Move UTXOs from the new federation into the old federation
+        // and clear the new federation's UTXOs
+        List<UTXO> utxosToMove = new ArrayList<>(provider.getNewFederationBtcUTXOs());
+        provider.getNewFederationBtcUTXOs().clear();
+        List<UTXO> oldFederationUTXOs = provider.getOldFederationBtcUTXOs();
+        oldFederationUTXOs.clear();
+        utxosToMove.forEach(utxo -> oldFederationUTXOs.add(utxo));
```

Comment:
prefer method references over lambdas

Reply:
This can be replaced with `oldFederationUTXOs.addAll(utxosToMove);`